src/desktop/apps/home/index.coffee

```diff
@@ -15,4 +16,6 @@ app.locals.markdown = markdown
 app.get '/', routes.index
 app.get '/log_in', routes.redirectLoggedInHome, routes.index
 app.get '/sign_up', routes.redirectLoggedInHome, routes.index
-app.get '/forgot', routes.index
+# TODO: Remove after /auth2 launches
+if !sd.NEW_AUTH_MODAL
+  app.get '/forgot', routes.index
```

Comment:
Hmm this too, we probably don't need. I guess we didn't end up needing much for this and it was probably just the path that was incorrect 🤔

Reply:
The routing here is from `/home` rather than `/auth` so it isn't covered by the conditional in the main routes.

Reply:
Yup yup, that's correct! Thanks. This looks good to merge.